sources/packages/web/src/services/http/dto/Note.dto.ts

```diff
@@ -7,7 +7,10 @@ export interface NoteAPIInDTO {
   noteType: StudentNoteType;
   description: string;
 }
-
+export interface NoteTypeItemsDTO {
```

Comment:
Please add blank lines before and after the method.
Why is this one not following the API naming conventions? Is it a GAP from the reorganization?

Reply:
I believe it must follow the same optionItemDTO that we have currently with label and value. Vue does not force us to use title and value keys only